Shared/Samples/Create buffers around points/CreateBuffersAroundPointsView.swift

```diff
@@ -19,6 +19,9 @@ struct CreateBuffersAroundPointsView: View {
     /// The view model for the sample.
     @StateObject private var model = Model()
     
+    /// A Boolean value indicating whether the settings view is showing.
+    @State private var showSettings = false
```

Comment:
Remember that Boolean identifier should read like an assertion, whereas this reads like a method name. A better name would be `settingsAreVisible`.